### PR TITLE
Add ComputationClient metrics to PjRtComputationClient

### PR DESCRIPTION
**Pull request by jonb377**
This change adds ComputationClient metrics to functions which have been implemented in PjRtComputationClient.

### CODE REVIEW

**Comment by JackCaoG** [comment]
rebase should fix the build issue

File: third_party/xla_client/pjrt_computation_client.cc [hunk 1]
```diff
@@ -114,31 +117,38 @@ std::vector<ComputationClient::DataPtr> PjRtComputationClient::TransferToServer(
         std::make_shared<PjRtData>(tensor.device, tensor.shape, buffer);
     datas.push_back(data);
   }
+  OutboundDataMetric()->AddSample(total_size);
+  CreateDataHandlesCounter()->AddValue(datas.size());
```
Comment (jonb377) [inline third_party/xla_client/pjrt_computation_client.cc:121]: @JackCaoG It looks like PjRt doesn't have the same notion of "handles" as XRT. I'm not familiar with handles in XRT, so I wanted to get some clarification - do these "handle" metrics still make sense to use in PjRt, or should we make a new counter?

Comment (JackCaoG) [reply third_party/xla_client/pjrt_computation_client.cc:121]: yea handle is same as buffer in the `TransferToServer` context.

**Review by will-cromar — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**